src/platform/intel/cavs/memory.c

```diff
@@ -105,7 +105,7 @@ struct mm memmap = {
 		.size = HEAP_RUNTIME_SIZE,
 		.info = {.free = HEAP_RUNTIME_SIZE,},
 		.caps = SOF_MEM_CAPS_RAM | SOF_MEM_CAPS_EXT |
-			SOF_MEM_CAPS_CACHE,
+			SOF_MEM_CAPS_CACHE | SOF_MEM_CAPS_DMA,
```

Comment:
this feels wrong... Is SueCreek the only platform that needs this? Or are you using the wrong memory allocator?